src/helpers/credential.rs

```diff
@@ -389,7 +404,7 @@ impl<'a> Credential<'a> {
             issuance_date,
             credential_subject,
             credential_schema,
-            credential_status,
+            credential_status: Some(credential_status),
```

Comment:
Shouldnt here the properties of the function also optional then?

Reply:
done, made it optional , changed bindings , added tests